modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages.
+
+//lahinson - oct. 2023 - The following paragraph was in Cesar's original draft. Does it still apply?
+
+When you enable monitoring dashboards, for each hosted cluster that the HyperShift Operator manages, the Operator creates a config map named `cp-[NAMESPACE]-[NAME]` in the `openshift-config-managed` namespace, where `NAMESPACE` is the namespace of the hosted cluster and `NAME` is the name of the hosted cluster. 
+
+[#hosted-control-planes-enable-dashboard]
+== Enabling monitoring dashboards
+
+To enable monitoring dashboards in a hosted cluster, complete the following steps:
+
+. Create the `monitoring-dashboard-template` config map in the `hypershift` namespace, being sure to specify the `--monitoring-dashboards` flag. For example:
```

Comment:
Create the `hypershift-operator-install-flags` configmap in the `local-cluster` namespace to specify `--monitoring-dashboards` in the `data.installFlagsToAdd`. For example: